Makefile

```diff
@@ -139,3 +139,7 @@ docker-update-dependencies: docker-build
 .PHONY: docs
 docs:
 	GO111MODULE=on go run -mod=$(MOD) -buildmode=exe cli/cmd/document/main.go $(DOCS_DIR)
+
+.PHONY: build-and-publish-release
+build-and-publish-release:
+	bash -c 'goreleaser --release-notes <(./scripts/print-version-changelog.sh)'
```

Comment:
Use `build-and-publish-release:SHELL=/bin/bash` instead of `bash -c`.

Reply:
TIL, will use.